src/HelloWorld.java

```diff
@@ -0,0 +1,5 @@
+public class HelloWorld {
+    public static void main(String ...args) {
+        System.out.println("Welcome to Linked In!");

```

Comment:
Does not match task specification
```suggestion
        System.out.println("Java: version 11.");
```